connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -101,7 +101,8 @@ class SparkConnectPlanner(
     throw new IllegalArgumentException("executeHolder does not belong to sessionHolder")
   }
 
-  private[connect] def session: SparkSession = sessionHolder.session
+  @DeveloperApi
+  def session: SparkSession = sessionHolder.session
```

Comment:
I am fine with this for now but I really think we should probably make `SparkConnectPlanner` an API first